test/tests.js

```diff
@@ -144,11 +177,11 @@ describe('XYZ Forwarder', function () {
         //         category: 'category'
         //     }
         // });
-        //
+        
         // window.MockXYZForwarder.eventProperties[0].label.should.equal('label');
         // window.MockXYZForwarder.eventProperties[0].value.should.equal(200);
 
-        done();
+        // done();
```

Comment:
Can we just delete this?

Reply:
i pushed so that it doesn't show up as being commented. shouldn't have touched this.

it's more of a call out in case partners need to test something that is async. keeping it doesn't hurt, but i think removing it from the function argument and here removes the potential call out